docs/job-server.md

```diff
@@ -1,21 +1,17 @@
 ## Running your code on the server
 
-To run code for real in the production environment, use the [https://jobs.opensafely.org](https://jobs.opensafely.org) site.
+To run code for real in the production environment, use [the jobs site](https://jobs.opensafely.org).
 Here you can see (even without a login) all the ongoing projects within OpenSAFELY, and the specific _jobs_ that have been run on the server.
 To submit job-requests (i.e., to run one or more actions), the general process is as follows:
 
 * **Log in** using your GitHub credentials
-* **Create a project** (or select an existing one):
-    * Click the `Add a New Workspace` button.
-    * Click the `Register Project` button.
-    * Choose a name, this will be the public facing name for your project.
-    * Click `Save`.
 * **Create a workspace**:
-    * Click the `Create a Workspace` button.
+    * Click the `Create a new workspace` button.
```

Comment:
The button on the home page currently says "Add a New Workspace"

Reply:
Fixed in companion PR: https://github.com/opensafely-core/job-server/pull/1942 (which I should have linked to in this one!)